app/data/mysql/migrations.go

```diff
@@ -61,6 +62,18 @@ func createOauthAccounts(db *sqlx.DB) error {
 	return err
 }
 
+func addOauthAccountEmail(db *sqlx.DB) error {
+	_, err := db.Exec(`
+		ALTER TABLE oauth_accounts ADD COLUMN email VARCHAR(255) DEFAULT NULL;
```

Comment:
Storing email like this kind of feels "off" to me, like this is the kind of thing oauth was supposed to prevent almost.  As a consumer of the oauth provider feels like we should be all set once we have a provider ID, and the email is really only needed so we can assign a username.

Is there a use case you had in mind for this beyond including more detail in the list of providers that comes back from the account / info endpoints?  It seems like just the configured provider names should be enough for any I can think of off hand.  I would lean towards simplifying in that way at least for this PR.  Adding columns causes runtime errors for updating users in production if they don't migrate.

If there is a use case you had in mind this could be done as its own PR - this one is already pretty big, and if we make this change I feel like it opens up a need to efficiently update emails when pre-existing oauth users log in and maybe some other stuff I haven't thought of.

Reply:
@AlexCuse that's a good point! While I understand your concerns about storing email addresses and the potential complexity it adds, there are some compelling reasons to include the user's email in the SQL table:

User Identification: While OAuth provides us with a provider ID, the email address is a more human-readable identifier. It can be useful in administrative tasks, and customer support.

Communication: Having the user's email address readily available in our database allows us to easily communicate with our users.

User Experience: If a user forgets which OAuth provider they used to sign up, having their email stored can help us guide them to the correct provider.

Reply:
@AlexCuse I have implemented you suggestion 😄